Classes/FileStorage/FileStorage.php

```diff
@@ -157,4 +159,22 @@ public function createFolder(string $folderIdentifier): void
             }
         }
     }
+
+    public function getTempPath(): string
+    {
+        return Environment::getVarPath() . '/transient/';
+    }
+
+    public function writeTempFile(string $filePrefix = '', string $fileContent = '', string $fileSuffix = ''): string|bool
```

Comment:
@mkarulin You should be able to use the return type string|false, since the method will never return true, if I'm not mistaken.

Reply:
Would have to be changed in the `core` package as well.